osx/setup-runtime

```diff
@@ -35,13 +35,13 @@ SKIP_LIBFFI=false
 FORCE_LIBYAML=false
 SKIP_LIBYAML=false
 FORCE_SQLITE3=false
-SKIP_SQLITE3=false
+SKIP_SQLITE3=true
```

Comment:
Line 38, 42 and 44 appear to be debugging changes that shouldn't be committed. They should be set back to false.